### PR TITLE
improved lighthouse score on blog's page

### PR DESCRIPTION
**Pull request by yash6318**
## Description
In this PR:

Addressed all the issues related to lighthouse improvement of the blog's page:

1. Added `alt` attributes to img tag.
2. Heading elements are now in sequentially-descending order.
3. Resolved the issue of link to a specific blog was not operational for the whole card.
4. Added some visual enhancement to the cardsfor better visibility  :upside_down_face: .
5. Improved `Links do not have discernible names`.
6. Improved CLS to the permitted range(for mobile).

Current Lighthouse:
image
- [Third-party Lighthouse Report (3G Connection)](https://www.webpagetest.org/result/221004_BiDcK5_DV7/)

<br>

After changes:
image

- [Third-party Lighthouse Report (3G Connection)](https://www.webpagetest.org/result/221004_AiDc2X_AB2/)

<br>

**_However there is some drop in the lighthouse score due to 'netlify' in the deployment. These can be ignored (for obvious reason)._**

<br>

PS: I guess there's something wrong in `best practices` in Lighthouse...yesterday the score of the current site was 83 in that but now showing 100 yet netlify deployment still shows the same i.e. 83. (It's because of the reason that  `errors logged in the console` but i have'nt changed anything related to this in this PR). 

## Fixes

Fixes part of #107 by @somudas

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [x] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

File: content/members/members.json [hunk 1]
```diff
@@ -411,5 +411,12 @@
     "github": "https://github.com/PaarthAgarwal",
     "link": "https://www.linkedin.com/in/paarth-agarwal-2291911b8/",
     "year": 2020
+  },
```
Comment (IamEzio) [inline content/members/members.json:414]: Remove this change. Will add members' data later.

**Review by IamEzio — CHANGES_REQUESTED** [review]
Thanks @yash6318. However, the issue description states much more that needs to be done in the blogs' page. PTAL

File: content/members/members.json (hunk 1 above)
Comment (yash6318) [reply content/members/members.json]: Done!

**Comment by yash6318** [comment]
> Thanks @yash6318. However, the issue description states much more that needs to be done in the blogs' page. PTAL

Sure! , will do in a while..

**Review by IamEzio — APPROVED** [review]

**Pull request merged**